### PR TITLE
Add-edit-and-remove

### PR DESCRIPTION
**Pull request by MostafaShahat2021**
### In this project :
- Add a function for adding a new task.
- Add a function for deleting a task.
- Add a function for editing task descriptions.
- Use webpack to bundle JavaScript.
- Use NPM.
- Use webpack.
- Use webpack-cli.
- Use css-loader.
- Use style-loader.
- Modify webpack.config.js
- use HtmlWebpackPlugin.
- Setup local dev server
- Add Linters for HTML/CSS.
- Use best practices for HTML & CSS.
- Update README file.

### CODE REVIEW

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @MostafaShahat2021 🖖 👏 , 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
-Professional ReadeMe documentation :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- All functions working well ✔️ 

## Required Changes ♻️
- [x] _Kindly check the comments under the review._

## Optional suggestions
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+// Delete Function
+window.del = (id) => {
+  tasks = tasks.filter((task) => task.index !== id);
+  tasks.forEach((e, i) => {
+    e.index = i + 1;
+  });
+  localStorage.setItem('task-list', JSON.stringify(tasks));
+  addElemToPage();
+};
```
[the remark this answers is not in the document]
Comment (MostafaShahat2021) [reply src/index.js]: Hi @AbayomiOlaoye, 
Thanks a lot for your efforts in reviewing my code 💯, I really appreciated it 😄.

[the remark this answers is not in the document]
Comment (AbayomiOlaoye) [reply src/index.js]: You know, very few understand that feedback is a gift. I'm really impressed and amazed by your calmness and expression of understanding. You didn't see the request as a threat like some people do. Thank you for your warm reception, and the kind words. They mean a lot to me 🤭👍

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @MostafaShahat2021 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MostafaShahat2021** [comment]
Hi @codepantha, Thanks a lot for your efforts in reviewing my code 😄 .

**Pull request merged**